### ISSUE TITLE
Web app doesn't render

### ISSUE THREAD

**Issue by jly36963** [wireapp/wire-webapp#8011]
## Wire for web version

`<meta property="wire:version" version="2020-01-06-09-57">`

## Browser version

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:71.0) Gecko/20100101 Firefox/71.0

## Installed browser addons

None

## Operating system

Windows 10 Pro 1909

## What steps will reproduce the problem?

* go to `app.wire.com`
* see 'transferring data from app.wire.com' in bottom left corner
* app never loads, blank white screen only. (No loading bar like it usually has.)

## What is the expected result?

* see loading bar
* app renders after short time

## What is the actual result?

* blank white screen, content never renders.

I've cleared my cache and told Firefox to "forget this site" to see if that would fix it, but it didn't.

**Comment by gopherit** [comment]
I experienced this several weeks ago where both instances of Wire for Windows (which uses Wire for Web) had a blank white screen, and app.wire.com wouldn't load either.

I experienced and am experiencing the exact same thing today, January 22, 2020, which just so happens to be the date of the new version of Wire for Web:  Wire for Web Version 2020.01.22.1255.  This issue doesn't appear to affect Wire for Android or iOS, which I assume don't use Wire for Web.

What is the root cause for this?  Is this server side or client side?  The first time it happened it made me think I lost my entire conversation history.  It happened today so I did something else for a few hours.  Now Wire on my Windows 10 box is loading, but Wire on an older Win system is still the same white screen.

Edit:  I let the frozen other Win system stay on the white screen for about 10-15 minutes, and Wire finally loaded.  Is the reason for this that the first load requires transferring the new version locally, and the Wire server back-end is swamped with requests for the new Web version?
